docs/config-r4e-v1_0.md

```diff
@@ -0,0 +1,119 @@
+---
+title: RHEL for Edge v1.0.0
+parent: Configuration specifications
+nav_order: 149
```

Comment:
100-150 are already used by the OpenShift spec (`rhcos` is grouped with it for historical reasons).  Alphabetically, R4E should use 199 and 200.